index.md

```diff
@@ -1,277 +1,71 @@
->
-> 1. Data wrangling and processing  
-> * **Goals:**   
-> * **Instructor's skills:**  
-> * **Materials:**    
-> * *Repository:* <>  
->
-{: .solution}
---->
-
-<!---
-> ## Genomics Workshop with Pipeline Workflow and R (Under development)
-> 1. Introduction & Using cloud/cluster computing for genomics  
-> * **Goals:**   
-> * **Instructor's skills:**  
-> * **Materials:**    
-> * *Repository:* <>  
->
-> 1. Introduction to the command line 
-> * **Goals:**   
-> * **Instructor's skills:**  
-> * **Materials:**    
-> * *Repository:* <>  
->
-> 1. Project Organization and Management 
-> * **Goals:**   
-> * **Instructor's skills:**  
-> * **Materials:**    
-> * *Repository:* <>  
->
-> 1. Data wrangling and processing 
-> * **Goals:**   
-> * **Instructor's skills:**  
-> * **Materials:**    
-> * *Repository:* <>   
->
-> 1. R for data analysis and visualization   
-> * **Goals:**   
-> * **Instructor's skills:**  
-> * **Materials:**    
-> * *Repository:* <>   
->
-{: .solution}
---->
-
-## Teaching Platforms
-In its current form, the workshop can be run on pre-imaged AWS (Amazon Web Services) instances, Cyverse instances, or data & directories built on a local compute cluster. Contact us for information on other platforms that those listed below.
-
-> ## Platforms Details
->
-> #### Amazon instance for workshop
-> 
-> All the software and data used in the workshop is on an Amazon AMI (Amazon Machine Image).
-> 
-> If you want to run your instance of the server used for this workshop, launch a t2.medium instance in the **N. Virginia** region with AMI **ami-aab445c7**, available under "Community AMIs" in the Amazon EC2 Management Console. Information on how to launch an instance can be found on the [creating Amazon instances page](http://www.datacarpentry.org/cloud-genomics/02-logging-onto-cloud/)
->
-<!---
->
-> #### Cyverse instance for the workshop
->
->
-> #### Building the workshop on HPC
---->
->
-{: .solution}
+| Lesson    | Overview |
+| ------- | ---------- |
+| [Project organization and management](https://datacarpentry.github.io/organization-genomics/)  |  Learn how to structure your genomic metadata, understand read metadata returned from your sequencing facility, and access data on the NCBI short read archive (SRA) database. |
+| [Data analysis and visualization in R](http://www.datacarpentry.org/R-genomics/)  |  Learn to import data, manage different data types, calculate summary statistics and create basic plots using the R programming language.  |
+| [Introduction to the command line](https://datacarpentry.github.io/shell-genomics/) | Learn to navigate your file system, create, copy, move, and remove files and directories, and automate repetitive tasks using scripts and wildcards. |
+| [Introduction to cloud computing for genomics](https://datacarpentry.github.io/cloud-genomics/) | Learn how to launch your own AWS instance and how to transfer data between your local computer and your virtual machine. |
```

Comment:
Same here. Maybe we should switch cloud computing and command line, given workshop order

Reply:
Same response as above. This order reflects my understanding of the order of the workshops.